networking/routes/route-configuration.adoc

```diff
@@ -66,3 +66,8 @@ include::modules/nw-ingress-edge-route-default-certificate.adoc[leveloffset=+1]
 include::modules/nw-ingress-reencrypt-route-custom-cert.adoc[leveloffset=+1]
 
 include::modules/nw-router-configuring-dual-stack.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information on specifying an alternative cluster domain, see xref:../../networking/ingress-operator.adoc#configuring-ingress[Specifying an alternative cluster domain using the appsDomain option].
```

Comment:
```suggestion
* For more information about specifying an alternative cluster domain, see xref:../../networking/ingress-operator.adoc#configuring-ingress[Specifying an alternative cluster domain using the appsDomain option].
```
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage

Reply:
acked